release.json

```diff
@@ -82,4 +82,4 @@
     "dca-1.9.0": {
         "SECURITY_AGENT_POLICIES_VERSION": "v0.3"
     }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
The file changes are generated by script, I will check if this can be improved so we don't have to deal with that last-line problem all the time.